js_modules/dagster-ui/packages/ui-core/src/gantt/GanttChart.tsx

```diff
@@ -441,118 +455,121 @@ interface GanttChartViewportContentsProps {
   onClickStep: (step: string, evt: React.MouseEvent<any>) => void;
 }
 
-const GanttChartViewportContents = (props: GanttChartViewportContentsProps) => {
+const GanttChartViewportContents = React.memo((props: GanttChartViewportContentsProps) => {
```

Comment:
Does this pretty reliably skip extra renders? (I ask because of the long props list with the many object-looking prop values...)

Reply:
Yeah it does seem to

Reply:
(Also doesn't hurt and would be done automatically by React compiler once we start using that)